people/methods/makeConnection.js

```diff
@@ -0,0 +1,16 @@
+var mysql= require('mysql');
+
+var connect=()=>{
```

Comment:
what is the purpose for this if this is being called repeatedly in the application